### ISSUE TITLE
The accuracy of downstream tasks

### ISSUE THREAD

**Issue by jiyt17** [guilk/VLC#2]
Hello! We pretrain VLC for 200K steps and try to reproduce the results of downstream tasks. But our scores are about one point lower than scores reported in the paper. Is there any details we need pay attention to? May you release the pretrained VLC model weights?

**Comment by guilk** [comment]
Hello, we will release the pretrained models this week. Before that, which task are you evaluating the model on? If it is VQA, we are using 576*576 during finetuning.

**Comment by jiyt17** [comment]
Thank you! We evaluate on VQA2.0 and NLVR2. We used images of the right size. And we notice that lr_mult is 1 in NLVR2. Should it be 10 like in VQA2.0?

**Comment by guilk** [comment]
I upload our pre-trained models and the VQAv2 submission files. For VQA, we use both training and validation splits for training, the learning rate of 5e-4, 10 training epochs and the image size of 576*576. Following UNITER and ALBEF, we also use the overlapping training data from VG-VQA dataset but it did not yield a huge difference (~0.2%).

No, lr_mult is not used in our setting. As you can see from [here](https://github.com/guilk/VLC/blob/00b4f3412f131414855e1ba4d294cfa416095027/vlc/modules/vlc_utils.py#L307), we use a layer-wise learning decay.

You can try with the model trained with a large set on VQAv2 task. I test it several times and the results on VQA are stable.

**Comment by jiyt17** [comment]
OK，thank you！

**Comment by jiyt17** [comment]
Hello, I basically reproduced your results with the released weight (about 0.3% lower). I have a question about the pretrain dataset CC3M, which should have 31 arrows. But our downloaded images compose 30 arrows. When we pretrain models, there is a prompt: Error while read file idx 1640497 in conceptual_caption_train_0 -> broken data stream when reading image file, which means some images are broken in CC_0. Did you meet such situation?

**Comment by guilk** [comment]
Good to know. 

I upload my VG-VQA data preprocessing scripts in `data_process`. I think once you add the additional VG-VQA data to VQAv2 finetuning, the gap can be closed. 

Yes, there are some corrupted images in CC3M. I just remove the images if they cannot be loaded by PIL. I upload my scripts `check_valid_images.py` to `data_process` folder.

**Comment by jiyt17** [comment]
OK, I understand. Thank you!
